### ISSUE TITLE
Full year grades empty

### ISSUE THREAD

**Issue by MRakox** [bain3/pronotepy#116]
To reproduce the issue, run the following code with pronotepy [v2.4.0](https://github.com/bain3/pronotepy/releases/tag/v2.4.0)
```py
import pronotepy

# initialise the client
# Note: the address should be a direct one (like the one below) usually the address shown by your school just redirects
# you to the real one.
# Ex.: https://your-school.com/pronote/students <-- BAD
#      https://0000000a.index-education.net/pronote/eleve.html <-- GOOD
#      https://0000000a.index-education.net/pronote/eleve.html?login=true <-- ONLY IF YOU HAVE AN ENT AND YOU KNOW YOUR IDS, ELSE REFER TO ENT PART OF README

client = pronotepy.Client('https://demo.index-education.net/pronote/eleve.html',
                          username='demonstration',
                          password='pronotevs')

if client.logged_in:
    # get the all the periods (may return multiple types like trimesters and semesters but it doesn't really matter
    # the api will get it anyway)
    periods = client.periods
    for period in periods:
        print(period.name, '---', len(period.grades))

```
**Output :**
Trimestre 1 --- 46
Trimestre 2 --- 19
Trimestre 3 --- 36
[...]
Année continue --- **0**

**Excepted output :**
Trimestre 1 --- 46
Trimestre 2 --- 19
Trimestre 3 --- 36
[...]
Année continue --- **101**

I think the title of the issue speaks for itself... but I'll quickly explain the problem:
The list of grades over the year is always empty with the last version of the module. While the other periods (trimesters and semesters) have grades. I have made some tests and the start and end dates of the period are correct...

**EDIT :** The full year averages do not work either. However, the absences are working!

**Comment by bain3** [comment]
Hi, this doesn't look like a problem on our part. Does this work on older versions? We don't create these periods, we get them directly from pronote. Most pronotes have some hidden periods that aren't used, like semesters. Basically it depends if the school puts all the grades into the "Année continue" period. But from the looks of it they only use it for absences.

**Comment by Bapt5** [comment]
What bain3 says is correct. Most of the schools do not put the grades in continuous years so it is the same for the averages.

**Comment by MRakox** [comment]
Oh... tanks anyway!
There is something strange... I tried the same thing about a month ago and it worked without problem.
Maybe a change directly in Pronote :thinking:

**Comment by MRakox** [comment]
I just tried it and it works with version 2.1.0
The behavior is a bit strange but the grades of the period are not empty...

**Comment by Bapt5** [comment]
This is a problem i solved in #106 because before there was an error in the request that caused Pronote return all the notes stored in the database

**Comment by MRakox** [comment]
Tanks a lot!
I guess I can close this issue now...
